### ISSUE TITLE
 Problem with resourcepacks

### ISSUE THREAD

**Issue by DINEXXL** [enjarai/shared-resources#45]
The problem is that when the mod is enabled, any resourcepacks sets added to the game are lost after restarting the game.
Minecraft version: 1.20.4
Shared Resources: 1.8.3

**Comment by aquahonoredhi** [comment]
when i first installed the mod, i had to move all the shared files to the shared folder, as none of them were moved automatically. hope this helps

**Comment by enjarai** [comment]
Correct, you have to manually move the files after first installing.

**Comment by DINEXXL** [comment]
I have them moved to the global folder, BUT when I put them in Minecraft they disappear after quitting the game

**Comment by aquahonoredhi** [comment]
Yes, this mod makes it so that the files in the .minecraft folders that are linked do not matter and only the files in the shared directory do.
